pages.es/common/git-rebase.md

```diff
@@ -34,4 +34,4 @@
 
 - Resuelve automáticamente cualquier conflicto favoreciendo la versión de la rama en la que se esta trabajando (en este caso la palabra `theirs` tiene un significado invertido):
 
-`git rebase -X theirs {{master}}`
+`git rebase -X theirs {{branch_name}}`
```

Comment:
I'm wondering if we shouldn't translate "branch_name" to Spanish, as there's no need for it to be in English.

Reply:
You're definitely right, the token should be translated.

Reply:
```suggestion
`git rebase -X theirs {{base_branch}}`
```

Reply:
I don't understand your suggestion, @navarroaxel — can you clarify the reasoning for it?

Reply:
Yeah, we should definitely translate here. I just inserted it as a suggestion as I unfortunately don't know the language

Reply:
@navarroaxel would `{{nombre_de_rama}}` or simply `{{rama}}` work here?

Reply:
I think rama is OK:
```suggestion
`git rebase -X theirs {{rama}}`
```
but if we use `rama_de_reorganización` above, this is another option:
```suggestion
`git rebase -X theirs {{rama_de_reorganización}}`
```
If you now English `theirs {{rama}}` has a lot of sense, otherwise `theirs {{rama_de_reorganización}}` is better.